types/angular/index.d.ts

```diff
@@ -316,7 +316,7 @@ declare namespace angular {
         /**
          * this is necessary to be able to access the scoped attributes. it's not very elegant
          * because you have to use attrs['foo'] instead of attrs.foo but I don't know of a better way
-         * this should really be limited to return string but it creates this problem: http://stackoverflow.com/q/17201854/165656
+         * this should really be limited to return string but it creates this problem: https://stackoverflow.com/q/17201854/165656
```

Comment:
```suggestion
         * this should really be limited to return string but it creates this problem: http://stackoverflow.com/q/17201854/165656
```
revert this change for now, it runs tests for related packages for `angular`, and those are quite outdated. Types augmentation in 3rd party types fails under TS 4.5